avaframe/com4FlowPy/flowCore.py

```diff
@@ -408,13 +409,12 @@ def calculation(args):
         row_idx = row_list[startcell_idx]
         col_idx = col_list[startcell_idx]
         dem_ng = dem[row_idx - 1: row_idx + 2, col_idx - 1: col_idx + 2]  # neighbourhood DEM
-        if varUmaxBool:
+        if varUmaxBool and varUmaxArray is not None:  # the check if uMax > 0 is already done where u is calculated to zDelta
```

Comment:
is the check that ```varUmaxArray[row_idx, col_idx]``` really done in lines 191 ff. ?
```if np.any(varUmaxArray[release>0] > 0):``` only checks if at least one element in ```varUmaxArray[release>0] is > 0```, so there might still exist cases where ```varUmaxArray[row_idx, col_idx]  <= 0``` which should probably be handled. Otherwise line 193 should probably read ```if np.all(varUmaxArray[release>0] > 0):``` ?

Reply:
I've changed if ``np.any(varUmaxArray[release>0] > 0)``... to :   ``varUmaxArray[varUmaxArray>0] = varUmaxArray[varUmaxArray>0]**2 / 2 / 9.81``, so cells that have a suitable (>0) uMax are accounted, if they are <= 0 the default value is used